composer-dependencies.json

```diff
@@ -0,0 +1,32 @@
+{
+  "require": {
+    "php": ">=7.4",
+    "krokedil/support": "dev-develop-system-report-settings",
```

Comment:
Do we need to make changes to this package dependency after changes we have made there?

Reply:
This is handled in #3.